### PR TITLE
Remove node reboot on CI

### PR DESCRIPTION
**Pull request by dirkmueller**
The reboot has been there to replace kernel-default-base with
kernel-default, but we could probably live with kernel-default-base.

Lets try it out.

## Why is this PR needed?

Does it fix an issue? addresses a business case?

add a description and link to the issue if one exists.

Fixes https://github.com/SUSE/avant-garde/issues/1147

## What does this PR do?

please include a brief "management" technical overview (details are in the code)

## Anything else a reviewer needs to know?

Special test cases, manual steps, links to resources or anything else that could be helpful to the reviewer.

## Info for QA

This is info for QA so that they can validate this. This is **mandatory** if this PR fixes a bug.
If this is a new feature, a good description in "What does this PR do" may be enough.

### Related info

Info that can be relevant for QA:
* link to other PRs that should be merged together
* link to packages that should be released together
* upstream issues

### Status **BEFORE** applying the patch

How can we reproduce the issue? How can we see this issue? Please provide the steps and the prove
this issue is not fixed.

### Status **AFTER** applying the patch

How can we validate this issue is fixed? Please provide the steps and the prove this issue is fixed.

## Docs

If docs need to be updated, please add a link to a PR to https://github.com/SUSE/doc-caasp.
At the time of creating the issue, this PR can be work in progress (set its title to [WIP]),
but the documentation needs to be finalized before the PR can be merged. 

# Merge restrictions

(Please do not edit this)

We are in *v4-maintenance phase*, so we will restrict what can be merged to prevent unexpected surprises:

    What can be merged (merge criteria):
        2 approvals:
            1 developer: code is fine
            1 QA: QA is fine
        there is a PR for updating documentation (or a statement that this is not needed)

### CODE REVIEW

**Comment by thehejik** [comment]
We need `kernel-default` (and reboot) due https://bugzilla.suse.com/show_bug.cgi?id=1149605 - missing ceph client side modules in kernel-default-base. The fix will probably not land earlier than in SLE15-SP2.

It is required for testing compatibility of CaaSP with SES/ceph.

**Comment by ereslibre** [comment]
IIRC we were missing some modules on the JeOS image for cilium to work as well. Check:

- https://github.com/SUSE/avant-garde/issues/291
- https://github.com/SUSE/avant-garde/issues/490

Unsure if https://github.com/SUSE/avant-garde/issues/490#issuecomment-508071732 was addressed:

> The fixed kernel-default-base is available over official SCC updates channel for SLE-15-SP1 but not present in any official JeOS image yet.

**Comment by dirkmueller** [comment]
Part of the rationale for the PR was.to find out what breaks.

We are running against JeOS-QU1 which is basically all maintenance updates as of a few weeks ago. Chances are good that this is in. The main deal breaker is the missing ceph modules if we do need those like it was pointed out.

File: ci/infra/openstack/cloud-init/commands.tpl [hunk 1]
```diff
@@ -1 +0,0 @@
-  - [ zypper, -n, install, ${packages} ]
```
Comment (ereslibre) [inline ci/infra/openstack/cloud-init/commands.tpl]: By removing this we are also removing the installation of the internal CA, that's the errors shown on CI currently.

**Comment by jordimassaguerpla** [comment]
Hi!
New code has been merged in master regarding terraform 0.12. Please rebase your branch to take this changes in.
Happy hacking

**Pull request closed without merging**